website/src/components/Sortable/SortableItem.tsx

```diff
@@ -25,6 +25,8 @@ export const SortableItem = ({ children, id }: PropsWithChildren<{ id: number }>
       className={`grid grid-cols-[min-content_1fr] items-center rounded-lg shadow-md gap-x-2 p-2 ${themedClasses}`}
       ref={setNodeRef}
       style={style}
+      {...attributes}
+      {...listeners}
     >
       <Button justifyContent="center" variant="ghost" {...attributes} {...listeners}>
```

Comment:
the attributes could be removed from here right?

Reply:
Left them in for the styling of the mouse pointer... but could also take it out and style manually

Reply:
yeah that would be great.

Reply:
oh sorry, I did not literally mean the `attributes` object, I meant all dnd stuff, I fixed it quickly.

I think it looks good with the mouse drag.

I will check the `...` problem mentioned below.